lib/LoadDeploymentActions.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const _ = require('lodash');
+const NotImplemented = require('./errors').NotImplemented;
+const CONST = require('./constants');
+const config = require('./config');
+
+loadActionScripts(config.actions);
+process.exit(0);
+
+function loadActionScripts(actions) {
+  function ensureDirectoryExist(filePath) {
+    const dirname = path.dirname(filePath);
+    if (fs.existsSync(dirname)) {
+      return true;
+    }
+    ensureDirectoryExist(dirname);
+    fs.mkdirSync(dirname);
+  }
+
+  function writeActionFile(template, filePath) {
+    if (fs.existsSync(filePath)) {
+      console.log(`File ${filePath} already exist`);
+    } else {
+      console.log(`Creating new file ${filePath}`);
+      ensureDirectoryExist(filePath);
+      fs.writeFileSync(filePath, Buffer.from(template, 'base64'), {
+        mode: CONST.FILE_PERMISSIONS.RWXR_XR_X
+      });
+    }
+  }
+
+  _.forEach(actions, (template, actionScriptName) => {
+    let extension = actionScriptName.split('.').pop();
+    if (extension === actionScriptName) {
+      writeActionFile(template, path.join(__dirname, 'fabrik/actions/sh', `${actionScriptName}`));
+    } else if (extension === 'js') {
+      writeActionFile(template, path.join(__dirname, 'fabrik/actions/js', `${actionScriptName}`));
+    } else {
+      throw new NotImplemented(`file type ${extension} is not supported`);
```

Comment:
rename the mesage instead to 'action hook defined in the file ${actionScriptName} has unsupported file extension. Only shell scripts (without extensions) & java scripts (with JS extensions) are supported'

- Having a vague message such as above makes it diff to debug for someone new.